components/earnings/commons/earnings_base.lua

```diff
@@ -56,15 +59,17 @@ function Earnings.calculateForPlayer(args)
 		error('"playerPositionLimit" has to be >= 1')
 	end
 
-	local playerConditions = '([[participant::' .. player .. ']] OR [[participant::' .. playerAsPageName .. ']]'
-		.. ' OR [[participantlink::' .. player .. ']] OR [[participantlink::' .. playerAsPageName .. ']]'
+	local playerConditions = {
+		'[[opponentname::' .. player .. ']]',
+		'[[opponentname::' .. playerAsPageName .. ']]',
+	}
 	for playerIndex = 1, playerPositionLimit do
-		playerConditions = playerConditions .. ' OR [[players_' .. prefix .. playerIndex .. '::' .. player .. ']]'
-		playerConditions = playerConditions .. ' OR [[players_' .. prefix .. playerIndex .. '::' .. playerAsPageName .. ']]'
+		table.insert(playerConditions, '[[opponentplayers_' .. prefix .. playerIndex .. '::' .. player .. ']]')
+		table.insert(playerConditions, '[[opponentplayers_' .. prefix .. playerIndex .. '::' .. playerAsPageName .. ']]')
 	end
-	playerConditions = playerConditions .. ')'
+	playerConditions = '(' .. table.concat(playerConditions, ' OR ') .. ')'
```

Comment:
Might as well switch to Module:Condition (might be OOS)

Reply:
imo oos for now